gridfinity-rebuilt-utility.scad

```diff
@@ -72,10 +72,19 @@ function height (z,d=0,l=0,enable_zsnap=true) =
 //          set n_div values to 0 for a solid bin
 // style_tab:   tab style for all compartments. see cut()
 // scoop_weight:    scoop toggle for all compartments. see cut()
-module cutEqual(n_divx=1, n_divy=1, style_tab=1, scoop_weight=1) {
+module cutEqual(n_divx=1, n_divy=1, style_tab=1, scoop_weight=1, place_tab=1) {
```

Comment:
Please add a @param statement for this parameter.

I know you explained it in gridfinity-rebuilt-bins, but that's not readily apparent to someone just reading the utility file or trying to understand the function.

Reply:
Added. I tried to mimic the existing documentation; let me know if more detail is needed.

Reply:
Good enough.